app/views/layouts/_navbar.html.erb

```diff
@@ -74,9 +72,20 @@
         </button>
       </form>
       <span>
-        <a href="#" class="desktop-only text-white"><i class="bi bi-bell-fill fs-3"></i></a>
+        <a href="#" class="desktop-only text-white" type="button" data-bs-toggle="offcanvas" data-bs-target="#notification-inbox-container"
+           aria-controls="notification-inbox-container"><i class="bi bi-bell-fill fs-3"></i></a>
         <a href="<%= profiles_me_path %>" class="desktop-only text-white"><i class="bi bi-person-fill fs-3"></i></a>
       </span>
     </div>
   </div>
-</nav>
\ No newline at end of file
+</nav>
+
+<div class="offcanvas offcanvas-end" data-bs-scroll="true" tabindex="-1" id="notification-inbox-container" aria-labelledby="notification-inbox-containerLabel">
+  <div class="offcanvas-header">
+    <h5 class="offcanvas-title" id="notification-inbox-containerLabel">Notifications</h5>
```

Comment:
There should be an I18n fpr the Notifications lable.

Reply:
👍🏻

Reply:
The question remains, where in the config files for the locales we should add the translation? In the folder landing_page? Have @WilliamBrandt or @EliasNotFound some insight here?